### ISSUE TITLE
plotting/file/line_compare second combo ("line join") does not work on Win

### ISSUE THREAD

**Issue by fpliger** [bokeh/bokeh#3612]
Happens on Chrome and FF

**Comment by bryevdv** [comment]
Moving to `0.12` to at least re-evaluate prior to release

**Comment by bryevdv** [comment]
Working now in both Chrome and FF (Win7 tested)

**Comment by bryevdv** [comment]
Spoke too soon. Works on FF, does not display WebGL on Chrome.

**Comment by bryevdv** [comment]
ping @almarklein

**Comment by almarklein** [comment]
The miters on the line on the left are indeed not drawn. It looks like this is due to the fact that the y-axis has a larger scale than the x-axis, and the code related to the miter cut-off does not deal with this anisotropy very well.

The algorithm was taken from code by Nicolas Rougier, and I adapted it to deal with anisotropic coordinate frames, but apparently I failed to make it completely invariant to it. To fix this, it would need significant redesign, and considering the complexity of this code, I expect this to take a lot of work.

Therefore, and because most people will use round joins (the default) anyway, I propose to leave this as-is, at least for now.
